yarn-project/sequencer-client/src/index.ts

```diff
@@ -5,3 +5,6 @@ export * from './sequencer/index.js';
 
 // Used by the node to simulate public parts of transactions. Should these be moved to a shared library?
 export * from './global_variable_builder/index.js';
+
+// Used by the validator to build blocks - should be moved to a shared library.
```

Comment:
Should this be an issue, related to above?

Reply:
Ah, i can actually remove this as i ended up making a hook instead of making a library!